diff --git a/meson.build b/meson.build
index 7e345ba6..063eee75 100644
--- a/meson.build
+++ b/meson.build
@@ -219,7 +219,7 @@ systemd_feat = get_option('systemd-session')
 systemd = dependency('systemd', required: systemd_feat)
 if systemd.found()
   install_data('data/labwc-session.target',
-    install_dir: systemd.get_variable('systemduserunitdir'))
+    install_dir: get_option('datadir') / 'systemd/user')
 endif
 
 icons = ['labwc-symbolic.svg', 'labwc.svg']
